src/compute-from-cog.ts

```diff
@@ -22,14 +23,24 @@ export const computeFromCog = async (
   forceLegacyCompose: string
 ) => {
   // Temporary check
-  // Check if cog is part of the accepted cog list
+
+  // Build isDepAccepted
+  let numDep = cog.substring(0, 2);
+  if ((numDep === "97") || (numDep === "98")) {
+    numDep = cog.substring(0, 3);
+  }
+  const isDepAccepted = acceptedDepList.includes(numDep);
+
+  // Check if dep or cog is part of the accepted dep or cog list
   const isCogAccepted = acceptedCogList.includes(cog);
```

Comment:
Si le département est accepté, alors on peut directement accepter le `codeCommune` pour éviter de lancer une recherche inutile. 
```suggestion
  const isCogAccepted = isDepAccepted || acceptedCogList.includes(cog);
```

Tu en penses quoi ?

Reply:
Oui bonne idée, je viens de faire le changement